src/sparseml/modifiers/obcq/base.py

```diff
@@ -40,8 +41,8 @@ class SparseGPTModifier(Modifier):
         True saves on GPU memory
     :param prunen: N for N:M pruning
     :param prunem: M for N:M pruning
-    :param compress_layers: list of layer names to compress during OBCQ, or __ALL__ to
-        compress every layer in the model
+    :param compress_layers: list of layer names to compress during OBCQ, or ALL_TOKEN
```

Comment:
sorry a bit confusing, we want to use the literal for documentation since this is what users would actually put in a recipe if doing manually. ALL_TOKEN is just to standardize the code
```suggestion
    :param compress_layers: list of layer names to compress during OBCQ, or '__ALL__'
```